_posts/2018-12-12-Catching-the-waves-of-servant-leadership.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+authors: [michaela_broeckx]
+title: "Catching the waves of servant leadership"
+image: /img/catching-the-waves-of-servant-leadership/main-image.png
+tags: [Agile, AgileWorks, Leadership, Tribe, Servant Leadership, Enterprise Agility, Change, Agile Coaching]
+category: Agile
+comments: true
+---
+
+As human beings, we’ve always had a complicated relationship with change.
+On the one hand, we see it all around us and try to cope with the complexity of it, attempting to avoid or even sabotage it in order to hang on for dear life, 
+to what we have always known to be normal, and therefore believe it to be what’s best.
+On the other hand, it’s what allows us to hope and strive, what drives us towards a dream for a better future – and so we look for ways to form it, 
+to mould it, even manipulate it at times, thinking that whatever comes out of the change will be better and lasting.
+
+But does change have an outcome? 
+Or is it endlessly moving forward? 
+This continuous tension between coping and moulding makes our relationship with change a very personal one. 
+Every day we strive, all of us, in this global community, to find a moment of peace with ourselves and with the ways we try to manage this uncertainty and complexity.
+And sometimes, when the odds are right, in a cursory moment of clear thoughts, we discover that precious balance, and see the bigger picture. 
+That’s when we embrace change and sense the opportunity...
+
+Projecting this mechanism from the personal into the organisational area, the same elements are at play. 
```

Comment:
I feel like there is a "When" missing here. I.e. "When projecting this mech....", but it's just a feeling :)

Reply:
Done, thanks!